src/GeositeFramework/plugins/layer_selector_v2/layers.json

```diff
@@ -150,5 +150,43 @@
                 "downloadUrl": "http://www.tceq.texas.gov/assets/public/gis/exports/au_oysterwaters.zip"
             }
         ]
+    },
+    {
+        "displayName": "WMS Example Layers",
+        "server": {
+            "type": "wms",
+            "layerType": "dynamic",
+            "url": "http://sampleserver1.arcgisonline.com/ArcGIS/services/Specialty/",
+            "name": "ESRI_StatesCitiesRivers_USA"
+        },
+        "includeLayers": [
+            {
+                "name": "States"
```

Comment:
Don't know if this is important but no service ID appears next to States when I load the map service.

![image](https://cloud.githubusercontent.com/assets/43062/13267652/7253fa22-da4c-11e5-9f02-f208ae83bde6.png)